polkadot-protocol/onchain-governance/overview.md

```diff
@@ -1,4 +1,326 @@
+
+Additionally, an alternating timetable permitted voting on either a public referendum or a council motion every 28 days (or 7 days on Kusama), which led to careful deliberation but limited the number of proposals considered.
+
+**OpenGov** revises the mechanisms for day-to-day decision-making, enhancing the scope and agility of referenda outcomes. This change enables the system to handle a higher volume of collective decisions at any given time.
+
+The following sections cover the key features of **OpenGov** and highlight its main distinctions from previous governance versions. 
+
+!!!note
+    Familiarity with [**Governance V1**](https://wiki.polkadot.network/docs/learn/learn-governance){target=\_blank} will provide valuable context for understanding the need for and direction of OpenGov. 
+
+## OpenGov Summary
+
+In **Governance V1**, network upgrade decisions were managed collaboratively by active token holders and the Council. 
+
+Proposals, regardless of whether they originated from the public or the Council, ultimately required a referendum, allowing all token holders to participate in the decision-making process.
+
+The Council acted as a representative body for the public, overseeing the treasury and initiating legislation. 
+
+However, it was sometimes perceived as a centralized authority. To enhance decentralization, **OpenGov** introduces these primary changes:
+
+- **Transferring all Council responsibilities to the public** through a direct democracy voting system.
+- **Dissolving the Council** to eliminate the centralized decision-making body.
+- **Expanding delegation options** so users can transfer voting power to trusted community members in various ways.
+- **Replacing the Technical Committee** with the Polkadot Technical Fellowship, a broader, community-oriented group.
+
+The diagram below provides an overview of the structure and changes introduced with Polkadot OpenGov.
+
+![](/images/polkadot-protocol/onchan-governance/opengov-structure.webp)
+
+### Origins and Tracks
+
+In Polkadot OpenGov, **Origins** and **Tracks** are key components in the governance process:
+
+An **Origin** defines the level of privilege or authorization a proposal has. It determines who can submit a proposal and what kind of proposal it is. For example, a proposal could come from a Treasury Origin for treasury-related requests, or from a Council Origin, which requires Council approval before voting.
+
+A **Track** is the process or pipeline a proposal follows after submission. It outlines how the proposal is handled, including how long the voting period lasts and other procedural details. Each Origin is linked to a specific Track.
+
+!!!note
+    More on Tracks and Origins is explained on [this](link to Tracks and Origins page) page.
```

Comment:
What about these links? I don't get if the idea would be to link above or, to link to an external source